deploy/helm/sumologic/conf/traces/traces.conf

```diff
@@ -0,0 +1,14 @@
+<source>
+    @type zipkin
+    port 9411
+</source>
```

Comment:
Labeling required. As tracing is last configuration part, previous `<match **>` catches all tracing traffic

Reply:
cc @vsinghal13 , maybe we should update all of our sources to use labels to make clear distinction between logs, metrics, and traces. when we added the `logs.source.generic.conf`, we didn't take into consideration non-log data when we added the `<match **>`

Reply:
I created jira task for it: SUMO-130461